### PR TITLE
upgraded bpmn-js to 0.9.2

### PR DESCRIPTION
**Pull request by mhariri**

### CODE REVIEW

**Comment by mhariri** [comment]
P.S. sorry for the many whitespace changes

File: pom.xml [hunk 1]
```diff
@@ -11,14 +11,14 @@
   <packaging>jar</packaging>
   <groupId>org.webjars</groupId>
   <artifactId>bpmn-js</artifactId>
-  <version>0.8.0-2-SNAPSHOT</version>
+  <version>0.9.2-0</version>
```
Comment (jamesward) [inline pom.xml]: This should be `0.9.2-SNAPSHOT` since Maven does the actual release.

File: examples/viewer/pom.xml [hunk 2]
```diff
@@ -44,7 +44,7 @@
         <dependency>
             <groupId>org.webjars</groupId>
             <artifactId>bpmn-js</artifactId>
-            <version>0.8.0</version>
+            <version>0.9.2-0</version>
```
Comment (jamesward) [inline examples/viewer/pom.xml]: This should be `0.9.2`

**Comment by mhariri** [comment]
I've changed it to a range value so that:
1. I can test the examples  before the release
2. the examples continue to work after the release without change

Check if you like it.

**Pull request merged**